### ISSUE TITLE
Email notification for new comments

### ISSUE THREAD

**Issue by jensbuehl** [jensbuehl/aoe4-guides#66]

**Comment by WesHaze** [comment]
Is this being worked on @jensbuehl ?

**Comment by jensbuehl** [comment]
No, there are no notifications on new comments and no time to implement that in the short term. Next Feature will be support for the new civs. 

Which link is dead?
Are you referring to Email verification?

**Comment by jensbuehl** [comment]
Implemented here 9df386195a51d6bfbe9a114d03e609c59d440876
